packages/web/src/pages/private-key-exporter-page/PrivateKeyExporterPage.tsx

```diff
@@ -0,0 +1,385 @@
+      <Flex alignItems='center' gap='s'>
+        <IconLifeRing
+          size='m'
+          fill={color.neutral.n800}
+          css={{ fill: 'yellow' }}
+        />
+        <Text variant='title' css={{ fontSize: 24 }}>
+          {messages.needHelp}
+        </Text>
+      </Flex>
+      <Text variant='body' textAlign='left'>
+        {messages.featureIntended}&nbsp;
+        <TextLink
+          href={`mailto:${AUDIUS_SUPPORT_EMAIL}`}
+          target='_blank'
+          css={{ color: color.primary.p500 }}
+        >
+          {messages.support}
+        </TextLink>
+      </Text>
+    </Flex>
+  )
+}
+
+const AgreeAndContinue = () => {
+  const dispatch = useDispatch()
+  const [, setIsPrivateKeyExporterModalVisible] =
+    useModalState('PrivateKeyExporter')
+  const [agreed, setAgreed] = useState(false)
+  const bulletPoints = [
+    messages.iUnderstand,
+    messages.iAccept,
+    messages.iAcknowledge
+  ]
+  const handleCancel = useCallback(() => {
+    dispatch(pushRoute(SETTINGS_PAGE))
+  }, [dispatch])
+  const handleProceed = useCallback(() => {
+    setIsPrivateKeyExporterModalVisible(true)
+  }, [setIsPrivateKeyExporterModalVisible])
+  return (
+    <Flex direction='column' alignItems='flex-start' mv='l' pv='xl' gap='xl'>
+      <Text variant='title' css={{ fontSize: 24 }}>
+        {messages.agree}
+      </Text>
+      <Flex alignItems='flex-start' gap='xl'>
+        <Switch isOn={agreed} handleToggle={() => setAgreed(!agreed)} />
+        <Flex direction='column' gap='s'>
+          {bulletPoints.map((bulletPoint, i) => (
+            <Text
+              key={`bullet-point${i}`}
+              variant='body'
+              textAlign='left'
+              css={{
+                display: 'list-item',
+                listStyleType: 'disc',
+                listStylePosition: 'inside'
+              }}
+            >
+              {bulletPoint}
+            </Text>
+          ))}
+        </Flex>
+      </Flex>
+      <Flex gap='xl' mt='s'>
+        <Button variant='secondary' onClick={handleCancel}>
+          {messages.goBack}
+        </Button>
+        <Button
+          variant='destructive'
+          onClick={handleProceed}
+          disabled={!agreed}
+        >
+          {messages.proceed}
+        </Button>
+      </Flex>
+    </Flex>
+  )
+}
+
+export const PrivateKeyExporterPage = () => {
+  useRequiresAccount()
+  const record = useRecord()
+  const user = useSelector(getAccountUser) ?? undefined
+  const [hasViewed, setHasViewed] = useState(false)
+  useEffect(() => {
+    if (user && !hasViewed) {
+      setHasViewed(true)
+      record(
+        make(Name.EXPORT_PRIVATE_KEY_PAGE_VIEWED, {
+          handle: user.handle,
+          userId: user.user_id
+        })
+      )
+    }
+  }, [user, hasViewed, record])
+  return (
+    <Flex direction='column' css={{ width: '100%' }}>
+      <Header />
+      <Box
+        mt='3xl'
+        pt='xl'
+        ph='4xl'
+        backgroundColor='white'
```

Comment:
Figma mocks mention dark mode, just a reminder to check the other themes!

Reply:
thanks